components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Twin/src/Deploy/IoTHubSupervisorDeployment.cs

```diff
@@ -94,6 +94,10 @@ public sealed class IoTHubSupervisorDeployment : IHostProcess {
                     HostConfig = new {
                         Binds = new[] {
                             "/mount:/mount"
+                        },
+                        CapDrop = new[] {
```

Comment:
What about windows deployment case ? Wouldn't those changes be applicable for that case as well ?

Reply:
These are Linux capabilities. they don't apply to Windows

Reply:
I was thinking here about IoT Edge runtime running through [EFLOW](https://docs.microsoft.com/en-us/azure/iot-edge/iot-edge-for-linux-on-windows?view=iotedge-2018-06&preserve-view=true), in which case containers will be effectively running on top of a Linux kernel where those capabilities would still be applicable. Though people might still be using IoT Edge on Windows version 1809 which does support Windows containers where I guess those capabilities will not be available.

Reply:
I will add a separate PBI to tackle this when we add support of EFLOW. For now this should be good for Windows containers.

Reply:
As you said in EFLOW container will run in a Linux environment running on a VM in Windows. Then it's a Linux case